index.test-d.ts

```diff
@@ -12,4 +12,17 @@ expectType<elementReady.StoppablePromise<Element | undefined>>(promise);
 expectType<elementReady.StoppablePromise<HTMLDivElement | undefined>>(elementReady('div'));
 expectType<elementReady.StoppablePromise<SVGElement | undefined>>(elementReady('text'));
 
+
+const {stop} = elementReady.subscribe('.unicorn', e => null);
+elementReady.subscribe('.unicorn', e => null, {target: document});
+elementReady.subscribe('.unicorn', e => null, {target: document.documentElement});
+elementReady.subscribe('.unicorn', e => null, {stopOnDomReady: false});
+
+elementReady.subscribe('.unicorn', e => expectType<Element>(e));
+elementReady.subscribe('div', e => expectType<HTMLDivElement>(e));
+elementReady.subscribe('text', e => expectType<SVGElement>(e));
```

Comment:
And these callbacks should not be inline arrow functions. They don't return anything so it's clearer with normal arrow functions.

Reply:
To be clear, Sindre means this by "normal arrow functions":

```js
elementReady.subscribe('text', e => {
	expectType<SVGElement>(e);
});
```